data/campaigns/Eastern_Invasion/units/Human_Shadow_Lord.cfg

```diff
@@ -62,8 +62,8 @@
         icon=attacks/dark-missile.png
         type=arcane
         range=ranged
-        damage=12
-        number=2
+        damage=8
```

Comment:
maybe 9?

Reply:
9 or 10 closer to 10, 30% res difference is 20% hp difference.